lib/api/incoming_defaults.js

```diff
@@ -0,0 +1,28 @@
+export const getIncomingDefaults = ({ Location, Page = 0, PageSize = 100, Keyword }) => ({
+    method: 'get',
+    url: 'incomingdefaults',
+    params: { Location, Page, PageSize, Keyword },
```

Comment:
```suggestion
    params: { Location, Page, PageSize, Keyword }
```

Reply:
I leave trailing commas in all the objects. I thought this is good practice (for instance I read it's recommended in the google style guide) and they're supported from ECMAScript 5 on.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas